### PR TITLE
Event vars

### PR DESCRIPTION
**Pull request by Phlosioneer**
Copy most of the event script vars from pokeruby to pokeemerald. A few were switched around.

### CODE REVIEW

**Comment by huderlem** [comment]
Any reason you didn't do them all?  For example, I noticed `VAR_0x405E` is not `VAR_SOOTOPOLIS_STATE`, so I'm assuming there are others that were skipped?

**Comment by Phlosioneer** [comment]
I did all the ones that actually do the same thing in both versions. Any others were either unused in ruby, or did something completely different in emerald.

File: include/constants/vars.h [hunk 1]
```diff
@@ -121,101 +121,101 @@
...
-#define VAR_0x4074                 0x4074
+#define VAR_ROUTE121_STATE                 0x4074
 #define VAR_0x4075                 0x4075
 #define VAR_0x4076                 0x4076
 #define VAR_0x4077                 0x4077
 #define VAR_0x4078                 0x4078
 #define VAR_0x4079                 0x4079
 #define VAR_0x407A                 0x407A
-#define VAR_0x407B                 0x407B
+#define VAR_ROUTE128_STATE                 0x407B
```
Comment (huderlem) [inline include/constants/vars.h]: Please re-align these columns

**Comment by huderlem** [comment]
Address formatting comment and resolve merge conflicts, then this is good to merge.

**Comment by Phlosioneer** [comment]
Updated this PR.

**Pull request merged**